x-pack/platform/plugins/shared/cases/public/components/all_cases/selector_modal/all_cases_selector_modal.tsx

```diff
@@ -60,7 +61,9 @@ export const AllCasesSelectorModal = React.memo<AllCasesSelectorModalProps>(
           `}
         >
           <EuiModalHeader>
-            <EuiModalHeaderTitle>{i18n.SELECT_CASE_TITLE}</EuiModalHeaderTitle>
+            <EuiModalHeaderTitle id="all-cases-modal-title">
```

Comment:
💅 Even though there's a low chance for a conflict, consider using `useGeneratedHtmlId` here instead.